### ISSUE TITLE
Filter modules unavailable

### ISSUE THREAD

**Issue by Kurimizumi** [RainLoop/rainloop-webmail#1248]
**RainLoop version, browser, OS:**
Rainloop version: v1.10.4.183
Rainloop server OS: Ubuntu 16.04.1
Web server: Apache 2.4.18-2
Email servers: Postfix 3.1.0-3 and Dovecot 2.2.22-1 with Sieve 2.2.22-1
Browser: Firefox 49.0.2 and Chrome 54.0.2840.87 m
Browser OS: Windows 7 Home Edition 64 Bit and CentOS 7 (1610)

**Expected behavior and actual behavior:**
Expected behaviour: Can select filters including Discard, Reject, Forward, etc
Actual behaviour: Was working at first, but now only shows the "Discard" filter module. 

**Steps to reproduce the problem:**
I didn't really do much. It broke soon after I deleted one of my rules, however. Other than that, I can't really give much more information.

I tried replacing static/js/min/app.min.js with static/js/app.js, and commenting out the 'Discard' key in the FiltersAction object, but there was no change in the browser (despite the new js file loading in correctly from what I saw)

**Logs or screenshots:**
![8d8cff9a508e4167bc447a882c0437e2](https://cloud.githubusercontent.com/assets/1858562/20042203/babd3f12-a46d-11e6-9342-6319a4edd2fc.png)

**Comment by Kurimizumi** [comment]
The Ajax returned when making the request is:

```json
{
Result: {
  Modules: {
    date: false,
    markasread: false,
    moveto: false,
    redirect: false,
    regex: false,
    reject: false,
    relational: false,
    vacation: false
  }
}
```

I've only recently started using Rainloop and so I'm not familiar with the structure of the project. However, I realise that the Discard option is still available due to that not being a conditional (you check that each module is enabled apart from the Discard module), but I couldn't find where this data was from...

**Comment by Kurimizumi** [comment]
I have just managed to create a workaround. I think it's actually a problem with my dovecot server. What I did was just force it to advertise support for all of the modules (modules that Rainloop supports was found in the source). Then, I made sure that these were explicitly enabled in the managesieve config. I've tested this and it now seems to work, and so I believe it to be a problem with managesieve not advertising support for enabled plugins properly on my system.
